app/controllers/foreman_bootdisk/api/v2/disks_controller.rb

```diff
@@ -42,6 +42,19 @@ def host
           end
         end
 
+        api :GET, '/subnets/:subnet_id', N_('Download subnet image')
+        param :subnet_id, :identifier_dottable, :required => true
+        def subnet
+          id = params[:id]
+          subnet = Subnet.find(id) || render_message(_("Cannot find subnet with id %s") % id) && return
```

Comment:
Ok done, I've also found that we require `view_hosts` permission, but it is missing from the `Bootdisk access` role, so adding it as well as `view_subnets` there. I will add a migration to add these to existing role perhaps, opinion?